server/utilis/validation/updateUserValidation.js

```diff
@@ -0,0 +1,10 @@
+const joi = require('joi');
+
+const updateUserValidation = joi.object({
+  firstName: joi.string().alphanum().min(3).required(),
+  lastName: joi.string().alphanum().min(3).required(),
```

Comment:
- I think firstName and LastName shouldn't be required, the user can update anything in his profile.
- if you can add more field to update will be excellent like `country, city, and  alt_mobile`

Reply:
Done